legate/info.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2021-2022 NVIDIA Corporation
```

Comment:
This module name is too generic for something that is tied specifically to jupyter. If there's a chance that we might add more magics in the future, this could be `legate/jupyter.py` to collect them. It might also be reasonable to make it "private" e.g. `legate/_jupyter.py` since this is not something users would/should ever import themselves.